internal/config/config.go

```diff
@@ -0,0 +1,121 @@
+package config
+
+import (
+	"fmt"
+	"github.com/rs/zerolog"
+	"strings"
+	"time"
+
+	clowder "github.com/redhatinsights/app-common-go/pkg/api/v1"
+	"github.com/spf13/viper"
+)
+
+type DatabaseT struct {
+	Host        string
+	Port        uint16
+	Name        string
+	User        string
+	Password    string
+	MaxIdleTime time.Duration
+	MaxLifetime time.Duration
+	MaxOpenConn int
+	MaxIdleConn int
+	LogLevel    int
+}
+
+type LoggingT struct {
+	Level  int
+	Stdout bool
+}
+type CloudwatchT struct {
+	Enabled bool
+	Group   string
+	Stream  string
+}
+type FeatureFlagsT struct {
+	Environment string
+	ExitOnPanic bool
+}
+
+type GlobalConfigT struct {
+	Database DatabaseT
+	Logging LoggingT
+	Cloudwatch CloudwatchT
+	Features FeatureFlagsT
+}
+
+var config GlobalConfigT
+var Database *DatabaseT
+var Logging *LoggingT
+var Cloudwatch *CloudwatchT
+var Features *FeatureFlagsT
+
+func init() {
+	var err error
+
+	viper.SetDefault("logging.level", 1)
+	viper.SetDefault("database.host", "localhost")
+	viper.SetDefault("database.port", 5432)
+	viper.SetDefault("database.loglevel", 1)
+	viper.SetDefault("cloudwatch.enabled", false)
+
+	if clowder.IsClowderEnabled() {
+		fmt.Println("Loading Clowder configuration")
```

Comment:
I would prefer to to do any standard output in the init() functions, unless you insist. For errors you do panic and that is fine, but for success path I would prefer not to write anything.

Reply:
The only I'd like to keep is the one informing you there is no `local.yaml` I do not have better idea how to inform user it might be a good idea to do in devel.